trade-versions/v1/documentation/quick-reference/QUICK-REFERENCE.md

```
@@ -56,38 +56,38 @@ curl http://localhost:8073/actuator
-| POST | `/api/v1/trades/details` | Create new trade |
-| PUT | `/api/v1/trades/details/{tradeId}` | Update existing trade |
-| GET | `/api/v1/trades/filter` | Filter trades by criteria |
-| POST | `/api/v1/trades/details/batch` | Batch create/update trades |
+| GET | `/v1/trades/details/portfolio/{portfolioId}` | Get trades by portfolio |
+| POST | `/v1/trades/details` | Create new trade |
+| PUT | `/v1/trades/details/{tradeId}` | Update existing trade |
+| GET | `/v1/trades/filter` | Filter trades by criteria |
+| POST | `/v1/trades/details/batch` | Batch create/update trades |
 
 ### Calendar-Based Endpoints
 | Method | Endpoint | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)